filebeat/module/iis/access/test/test-x-forward-for.log-expected.json

```diff
@@ -205,8 +214,9 @@
         "service.type": "iis",
         "source.address": "192.168.7.63",
         "source.ip": "192.168.7.63",
+        "url.original": "/Production-UI/data/finance/legacy/GLATrnsDetail",
         "url.path": "/Production-UI/data/finance/legacy/GLATrnsDetail",
-        "url.query": "$filter=Subsys%20eq%20%27JE%27%20and%20Ref%20eq%20%27HSNG08-MR%27%20and%20BatchId%20eq%20%27FY21HSNG0820%27&$orderby=RecNo&$skip=0&$top=20",
+        "url.query": "$filter=Subsys eq 'JE' and Ref eq 'HSNG08-MR' and BatchId eq 'FY21HSNG0820'&$orderby=RecNo&$skip=0&$top=20",
```

Comment:
since this is a result of the switch to the `uri_parts` decoder, I'm fine with the url-decoded values, but I think we'll need to document that (potentially apart from `url.original`) the `url.*` fields are now all decoded since it's a breaking change.

Reply:
So if we want to say ALL `url.*` fields, we'd have to update the remining pipelines.  I only touched the ones that had sample data and could validate the change.  There were a bunch that had the "auto generated" JS pipelines from the rsa2elk that had actions to parse a url field but no sample data to show so I didn't touch those. I can go though and update those too if we want?

Reply:
Yeah, I'd be fine just documenting it as a single line item with a list of all the modules that were updated under the `filebeat` section in the changelog, but lest the wording needs to change, may want to hold off until there's consensus about `url.original`